utility_scripts/compare_models.py

```diff
@@ -97,16 +99,14 @@ def _create_argument_parser():
                  'must be supplied in either TSV or gzipped TSV format!'
         )
 
-        required.add_argument(
+        optional.add_argument(
             '-l2',
             '--label_file_model_2',
```

Comment:
According to [GNU](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html), long-options should use hyphens instead: `--label-file-model-2` (and max 3 words, though I guess the last is simply a number so let's maybe not worry too much about that)